src/AppInstallerCLITests/AppInstallerCLITests.vcxproj

```diff
@@ -337,6 +340,12 @@
     <CopyFileToFolders Include="TestData\Manifest-Bad-DuplicateKey.yaml">
       <DeploymentContent>true</DeploymentContent>
     </CopyFileToFolders>
+    <CopyFileToFolders Include="TestData\Manifest-Bad-DuplicateReturnCode-ExpectedCodes.yaml">

```

Comment:
>Manifest-Bad-DuplicateReturnCode-ExpectedCodes.yaml [](http://example.com/codeflow?start=41&length=51)

Looks like the new test files forgot tobe included in the commit causing test failure